### ISSUE TITLE
Tracing. Press any key to quit...

### ISSUE THREAD

**Issue by chongbo2013** [chame1eon/jnitrace#57]
Tracing. Press any key to quit...

**Comment by chame1eon** [comment]
Hi,

Please provide more information. Such as the application you are testing with, the version of Frida you are using and the version of jnitrace.

Thanks

**Comment by Th3-R3p4ck3r** [comment]
I try it on win 11 same issues latest python and pip and jnitrace 3.3.1 and previous one
With frida 16.0.11 and previous with more than one app and get empty json
I use it with ldplayer 5 android 7.1 32

**Comment by chame1eon** [comment]
Hi, have you been able to use other frida scripts with ldplayer 5? jnitrace has not been tested or used against ldplayer. Thanks

**Comment by Th3-R3p4ck3r** [comment]
I try it few months ago and it was working fine with ldplayer 
##########
Now working 
I figure it
If ldplayer 32 u need apk lib x86 if arm or other than it then not working

**Comment by ahmedmani** [comment]
jnitrace does not work on emulators that use arm translation even tho frida supports it

**Comment by cTFk1ller** [comment]
Yes, it won't work on Android Studio emulators or any emulators that use arm translation.

**Comment by ITmustang** [comment]
@chame1eon i am facing same error. 
~/…/ViewerJNI/resources/lib/arm64-v8a]
└─$ jnitrace -l libnative-lib.so com.example.viewerjni
ERROR: {'type': 'error', 'description': 'TypeError: not a function', 'stack': 'TypeError: not a function\n    at c (node_modules/jnitrace-engine/dist/engine.js:35)\n    at <anonymous> (node_modules/jnitrace-engine/dist/index.js:96)\n    at call (native)\n    at o (node_modules/browser-pack/_prelude.js:1)\n    at <anonymous> (node_modules/browser-pack/_prelude.js:1)\n    at <anonymous> (jnitrace/src/main.ts:8)\n    at call (native)\n    at o (node_modules/browser-pack/_prelude.js:1)\n    at r (node_modules/browser-pack/_prelude.js:1)\n    at <eval> (/script1.js:3811)', 'fileName': 'node_modules/jnitrace-engine/dist/engine.js', 'lineNumber': 35, 'columnNumber': 1}
Tracing. Press any key to quit...
im running a physical device (pixel5a) with android 14 with frida-17.2.16 on both the android phone and kali box with jnitrace 3.3.1

**Comment by Decap1tator** [comment]
> [@chame1eon](https://github.com/chame1eon) i am facing same error. ~/…/ViewerJNI/resources/lib/arm64-v8a] └─$ jnitrace -l libnative-lib.so com.example.viewerjni ERROR: {'type': 'error', 'description': 'TypeError: not a function', 'stack': 'TypeError: not a function\n at c (node_modules/jnitrace-engine/dist/engine.js:35)\n at (node_modules/jnitrace-engine/dist/index.js:96)\n at call (native)\n at o (node_modules/browser-pack/_prelude.js:1)\n at (node_modules/browser-pack/_prelude.js:1)\n at (jnitrace/src/main.ts:8)\n at call (native)\n at o (node_modules/browser-pack/_prelude.js:1)\n at r (node_modules/browser-pack/_prelude.js:1)\n at (/script1.js:3811)', 'fileName': 'node_modules/jnitrace-engine/dist/engine.js', 'lineNumber': 35, 'columnNumber': 1} Tracing. Press any key to quit... im running a physical device (pixel5a) with android 14 with frida-17.2.16 on both the android phone and kali box with jnitrace 3.3.1

I'm having this same error running jnitrace on a cuttlefish arm64 device.
